assets/css/pages/home/index.css

```diff
@@ -0,0 +1,26 @@
+@import './header.css';
+@import 'what_we_do.css';
+@import 'who_we_helped.css';
+@import 'how_we_do_it.css';
+@import 'what_we_use.css';
+@import 'lets_work_together.css';
+@import 'footer.css';
```

Comment:
This is nice! Somehow I've never seen this done before.

Reply:
Does this recursive calls for CSS file though, or does it render everything to a single file? It might be bad if the client needs to call the HTML, then the top level CSS, then the second level CSS.